kubernetes/prometheus.yaml

```diff
@@ -0,0 +1,493 @@
+---
```

Comment:
maybe this could live in a samples/ dir or similar?

i think we want to have a README with the chunk of config to add to an existing Prom config as the entry point.